hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/BaseRollbackPlanActionExecutor.java

```diff
@@ -115,7 +115,6 @@ protected Option<HoodieRollbackPlan> requestRollback(String startRollbackTime) {
           LOG.warn("Request Rollback found with instant time " + rollbackInstant + ", hence skipping scheduling rollback");
         } else {
           table.getActiveTimeline().saveToRollbackRequested(rollbackInstant, rollbackPlan);
-          table.getMetaClient().reloadActiveTimeline();
```

Comment:
I found all the invokers would realod the active timeline instantly right after the rollback, so this is redundant, please double check.